src/ci/resources/tes_application.conf.ctmpl

```diff
@@ -0,0 +1,76 @@
+include required(classpath("application.conf"))
+include "build_application.inc.conf"
+
+filesystems {
+    blob {
+        class = "cromwell.filesystems.blob.BlobPathBuilderFactory"
+        global {
+            class = "cromwell.filesystems.blob.BlobFileSystemManager"
+            config {
+                workspace-manager: {
+                    # static: Where is WSM Endpoint
+                    url: "https://workspace.dsde-dev.broadinstitute.org"
+                    # See comment in src/ci/resources/acquire_b2c_token.sh for the one-time setup required to automatically acquire this.
+                    # This token can also be manually acquired with gcloud auth print-access-token myAccount@gmail.com
+                    b2cToken: "{{ env "CROMWELL_B2C_TOKEN" }}"
+                }
+            }
+        }
+    }
+}
+
+engine {
+  filesystems {
+    local {
+      enabled: true
+    }
+    http {
+      enabled: true
+    }
+    blob {
+      enabled: true
+    }
+  }
+}
+
+backend {
+  default = "TES"
+  providers {
+    TES {
+      actor-factory = "cromwell.backend.impl.tes.TesBackendLifecycleActorFactory"
+      config {
+        # Root of a blob storage container for Cromwell to use
+        root = "https://lz813a3d637adefec2c6e88f.blob.core.windows.net/sc-d8143fd8-aa07-446d-9ba0-af72203f1794/workspace-services/cbas/terra-app-8dca2584-99fc-4fcb-8843-f7b6ad0302a2/"
```

Comment:
I think the `root`, `endpoint`, and `workflow-execution-identity` all need to be set by the user here and shouldn't be pre-populated. Out of scope for this ticket, but we could think about way to use Justin's script to generate these for us.